website/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.21 as synchronizer
+
+COPY ./website ./website
+COPY ./scripts ./scripts
+COPY Makefile .
+COPY .mdox.yaml .
+
+RUN go install github.com/bwplotka/mdox@latest
+RUN apt update && apt install -y git make
+
+RUN make synchronize
+
+FROM hugomods/hugo:exts as builder
+
+ARG HUGO_BASEURL=
+ENV HUGO_BASEURL=${HUGO_BASEURL}
+
+RUN ls -la
+
+COPY --from=synchronizer /go/website .
+
+RUN npm install --ignore-scripts
+
+RUN hugo --minify --gc --cleanDestinationDir
+
+FROM hugomods/hugo:nginx
+
+# Copy the generated files to keep the image as small as possible.
+COPY --from=builder /src/public /site
+
+RUN ls -la /site
```

Comment:
Are you sure you want to run this command when building the image ? That's a debug layer for me, probably we should remove it for "production".

Reply:
Ups!
YOu're right, I'll remove